src/geometry/rtdoc.hpp

```diff
@@ -33,6 +33,9 @@ struct RTDoc {
 
   [[nodiscard]] static rect_internal to_rect(poly_type const& poly) {
     const auto& points = poly.outer();
+    if (points.empty()) {
+      return rect_internal{};
```

Comment:
should presumably throw? this is an illegal polygon, no?

Reply:
Yes, this is just temporary to avoid a crash. There are more erroneous Polygons which are currently being accepted that we should look into.
https://github.com/RediSearch/RediSearch/pull/3337/files#diff-a20811939295824e42431eaab78364ae4fe5be42ed6b0b2d8f1b3509aa9bd105R60-R70 

The wkt parser is not so strict. We should look at [is_valid](https://www.boost.org/doc/libs/1_81_0/libs/geometry/doc/html/geometry/reference/algorithms/is_valid/is_valid_1.html) and [correct](https://www.boost.org/doc/libs/1_81_0/libs/geometry/doc/html/geometry/reference/algorithms/correct/correct_1.html).